src/components/SearchModal/styleds.tsx

```diff
@@ -155,3 +143,29 @@ export default function SearchInputWithIcon({
     </SearchInputWrapper>
   )
 }
+
+export const Checkbox = styled.input`
+  border: 1px solid ${({ theme }) => theme.red2};
+  height: 20px;
+  margin: 0;
+`
+
+export const TextDot = styled.div`
+  height: 3px;
+  width: 3px;
+  background-color: ${({ theme }) => theme.text2};
+  border-radius: 50%;
+`
+
+export const SeparatorDark = styled.div`
+  width: 100%;
+  height: 1px;
+  background-color: ${({ theme }) => theme.bg3};
+`
+
+export const GoBackIcon = styled(ChevronLeft)<{ onClick: () => void }>`
```

Comment:
If you wrap those icons in a button you won't need `onClick` prop here
```suggestion
export const GoBackIcon = styled(ChevronLeft)`
```